ts/features/fims/history/components/__tests__/FimsHistoryListItems.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { createStore } from "redux";
+import { ServicePublic } from "../../../../../../definitions/backend/ServicePublic";
+import { Access } from "../../../../../../definitions/fims_history/Access";
+import { applicationChangeState } from "../../../../../store/actions/application";
+import { appReducer } from "../../../../../store/reducers";
+import { renderScreenWithNavigationStoreContext } from "../../../../../utils/testWrapper";
+import {
+  FimsHistoryFailureListItem,
+  FimsHistorySuccessListItem
+} from "../FimsHistoryListItems";
+
+const mockServicePublic = { organization_name: "TEST" } as ServicePublic;
+const mockAccess: Access = {
+  id: "TESTING",
+  redirect: { display_name: "TESTING", uri: "TESTING" },
+  service_id: "TESTING_SID",
+  timestamp: new Date(0)
+};
+
+describe("FimsHistoryListItems", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it("should match snapshot for SuccessListItem", () => {
+    const component = renderComponent(
+      { consent: mockAccess, serviceData: mockServicePublic },
+      FimsHistorySuccessListItem
+    );
+
+    expect(component).toBeDefined();
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+  it("should match snapshot for FailureListItem", () => {
+    const component = renderComponent(
+      { item: mockAccess },
+      FimsHistoryFailureListItem
+    );
+
+    expect(component).toBeDefined();
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+});
+
+type ListItemProps =
```

Comment:
Instead of redefining the type(s), why do not we export the original ones (`SuccessListItemProps` and `BaseHistoryListItemProps`) and use them directly in `renderComponent` first input's type?